examples/access-token-server/status.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
This is required for tests.

Reply:
Possible solution

1. Add status.json to .gitignore
2. Create status.json.template
3. Modify test beforeAll to copy status.json.template to status.json